### DISCUSSION TITLE
Development progress of Stitches

### DISCUSSION THREAD

**Discussion by Razinsky** [stitchesjs/stitches#328] [Ideas]
Hi there!

I've been using Stitches for a couple months now for my business website. I'm on the latest canary version. Even though I absolutely love it, there are a couple annoying things and a lot of those seems to be fix in PRs that are waiting to be merged... 

Besides that, my biggest really issue is FOUC with the build.

Anyway, I just wanted to know if the project is still active since we haven't seen any release is a long while.

Thanks, keep up the good work, Stitches is really promising.

**Comment by peduarte** [comment]
hey, we've been busy working on the beta version of stitches. We've released a canary of it now, its for testers. But if youre interested in testing it, check out the migration guide https://stitches-site-git-beta.modulz-deploys.com/blog/migrating-from-alpha-to-beta
